src/libraries/System.Private.CoreLib/src/System/Threading/PortableThreadPool.GateThread.cs

```diff
@@ -90,7 +91,7 @@ private static void GateThreadStart()
                                     newCounts.NumThreadsGoal = newNumThreadsGoal;
 
                                     ThreadCounts oldCounts = threadPoolInstance._separated.counts.InterlockedCompareExchange(newCounts, counts);
-                                    if (oldCounts == counts)
+                                    if (oldCounts == counts && !OperatingSystem.IsBrowser())
```

Comment:
This also should have had `[UnsupportedOSPlatform("browser")]`, Overall this GateThread, WaitThread, WorkerThread all used in PortableThreadPool are could have `[UnsupportedOSPlatform("browser")]` which makes PortableThreadPool itself  `[UnsupportedOSPlatform("browser")]`, or they should excluded from browser build. Added this condition just until get proper direction

Reply:
The whole thread pool implementation isn't viable on browser.  Until browser has threads support, ThreadPool needs a browser-specific implementation.

Reply:
This warning was false positive, reverted